src/main/java/com/toptrumps/core/AIPlayer.java

```diff
@@ -1,4 +1,16 @@
 package com.toptrumps.core;
 
-public class AIPlayer {
-}
+
+import java.util.ArrayList;
+
+public class AIPlayer extends Player{
+
+    private String name;
+    private ArrayList<Card> hand;
+
+    public AIPlayer(String name){
```

Comment:
I think the instance variables would be better in the Player class then call super in the constructor here.

Reply:
Are you sure this will work? Each player in my mind has to have their own hand. If I make the class instance public or protected they'll share a hand, and it'll cause all sorts of issues down the line. In my mind 1 player should have 1 hand, and the hand will be used to see when they get eliminated. Ie. Empty hand? Out of the game.

Reply:
Yes they will have their own hand that will be created when we create instances of the object. These will be different for each instance we create - it's not a static attribute